types/fusebit__oauth-connector/index.d.ts

```diff
@@ -0,0 +1,162 @@
+// Type definitions for @fusebit/oauth-connector 1.3
+// Project: http://fusebit.io/
+// Definitions by: Andrew Miller <https://github.com/andrewrmiller>
+//                 Trent Hashimoto <https://github.com/hashiphoto>
+//                 Andy Dam <https://github.com/andydam>
```

Comment:
As in other thread, this requires explicit opt-in by those users, as DT sends automated notifications to package owners.

Reply:
I have requested that they each review this PR.